src/redux/migrations.ts

```diff
@@ -1861,4 +1861,23 @@ export const migrations = {
     ...state,
     account: _.omit(state.account, 'pictureUri'),
   }),
+  228: (state: any) => {
```

Comment:
can we add a unit test for this since this doesn't seem like a straightforward migration?

Reply:
unit test added in [e005b33](https://github.com/valora-inc/wallet/pull/5867/commits/e005b333701d71b6e2cac1c1aee0a27b640ee957)